python/instrumentation/openinference-instrumentation-litellm/tests/test_instrumentor.py

```diff
@@ -0,0 +1,174 @@
+import json
+
+import litellm
+import pytest
+from openinference.instrumentation.litellm import LiteLLMInstrumentor
+from openinference.semconv.trace import (
+    EmbeddingAttributes,
+    SpanAttributes,
+)
+from opentelemetry.sdk.resources import Resource
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import SimpleSpanProcessor
+from opentelemetry.sdk.trace.export.in_memory_span_exporter import InMemorySpanExporter
+
+
+@pytest.fixture(scope="class")
+def in_memory_span_exporter() -> InMemorySpanExporter:
+    return InMemorySpanExporter()
+
+
+@pytest.fixture(scope="class")
+def tracer_provider(in_memory_span_exporter: InMemorySpanExporter) -> TracerProvider:
+    resource = Resource(attributes={})
+    tracer_provider = TracerProvider(resource=resource)
+    tracer_provider.add_span_processor(SimpleSpanProcessor(in_memory_span_exporter))
+    return tracer_provider
+
+
+@pytest.fixture(scope="class")
+def instrumentor(tracer_provider: TracerProvider) -> LiteLLMInstrumentor:
+    instrumentor = LiteLLMInstrumentor(tracer_provider=tracer_provider)
+    instrumentor.instrument()
+    yield instrumentor
+    instrumentor.uninstrument()
+
+
+class TestLiteLLMInstrumentor:
+    def test_completion(tracer_provider, in_memory_span_exporter, instrumentor):
+        in_memory_span_exporter.clear()
+
+        litellm.completion(
+            model="gpt-3.5-turbo",
```

Comment:
are you somehow stubbing out the gpt-3.5-turbo call?

Reply:
otherwise if this test actually relies on a network connection out and an openAI key, this probably won't work

Reply:
might be worth taking a look at this
https://litellm.vercel.app/docs/completion/mock_requests

Reply:
Made all unit tests use mock responses so no more need for network connection or an openAI key